source/minimum-n.d.ts

```diff
@@ -0,0 +1,31 @@
+/**
+Create a type of one or more elements.
+
+@category Utilities
```

Comment:
```suggestion
@category Array
```
Categories were made more specific in #331